### DISCUSSION TITLE
What is a farming event?

### DISCUSSION THREAD

**Discussion by brianwfreeman** [martomi/chiadog#85] [Q&A]
I am getting the following warning:
"Last farming event was 34 seconds ago. This is unusual."
What would cause a slow farming event? What is a farming event?

**Comment by martomi** [comment]
Farming event in this context is when your farmer responds to a challenge to a signage point. Usually signage points are processed every ~9 seconds. Sometimes there are time gaps up to 60 seconds and that's unusual but normal. The more important metric is that no signage points are skipped - and we check and notify for such events. There's no notification for the event you mention because it's just an interesting observation - not a problem.
